### ISSUE TITLE
Windows Server 2016 / SQL Server 2016

### ISSUE THREAD

**Issue by irongeekger** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#635]
Hi,

please update this article according Windows Server 2016 Support in CU6 and SQL Server 2016 Support in CU7.

---
#### Dokumentdetails

⚠ *Bearbeiten Sie diesen Abschnitt nicht. Er ist für die Verknüpfung von docs.microsoft.com zum GitHub-Artikel erforderlich.*

* ID: 7c475b5d-09ed-7f58-86d4-27f28e33ef7a
* Version Independent ID: f0ad50f3-ea93-9424-6de8-a0b778e90b23
* Content: [Serveranforderungen für Skype for Business Server 2015](https://docs.microsoft.com/de-de/skypeforbusiness/plan-your-deployment/requirements-for-your-environment/server-requirements#feedback)
* Content Source: [Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness-pr.de-DE/blob/live-sxs/Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md)
* Product: **skype-for-business-itpro**
* GitHub Login: @MicrosoftHeidi
* Microsoft Alias: **heidip**

**Comment by kenwith** [comment]
@irongeekger - @<alias> - Thank you for submitting this feedback. And thanks for the engagement @JeffBrownTech. 

This is a good catch now that CU7 is out. I was able to get this into the backlog for the Skype team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
